app/components/custom-controls.js

```diff
@@ -0,0 +1,60 @@
+import Component from '@ember/component';
+import { argument } from '@ember-decorators/argument';
+import { action, computed } from '@ember-decorators/object';
+import { run } from '@ember/runloop';
+
+export default class ShareControls extends Component {
+  // Share Control properties
+  @argument shareURL;
+  @argument shareClosed = true;
+  @argument copySuccess = false;
+
+  // Data Control properties
+  @argument dataClosed = true;
+  @argument boundsGeoJSON;
+
+  datasets = [
```

Comment:
Should this also be an `@argument` with this default?

Reply:
Class updated. 

What change are you proposing to the argument?